cranelift/jit/src/backend.rs

```diff
@@ -191,6 +191,8 @@ pub struct JITModule {
     pending_got_updates: Vec<GotUpdate>,
 }
 
+unsafe impl Send for JITModule {}
```

Comment:
Instead of implementing `Send` on `JITModule`, which may cause issues if we add a new non-`Send` field, maybe it would be possible to implement `Send` on each type that doesn't currently like `CompiledBlob` and `PtrLen`. For the `symbols` field you can for example wrap the `*const u8` in a newtype.

Reply:
Should be mostly good now I think. I made a `SendWrapper` for fields which can be wrapped in it in `JITModule`, and impl'd `Send` on any types that didn't like `CompiledBlob` and `PtrLen`. I also had to wrap all the `NonNull` fields as they were also offenders